# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../moneypenny/datadir.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./src/globals.cpp	2006-02-04 10:40:46.000000000 +0100
+++ ./src/globals.cpp	2006-03-16 11:41:55.000000000 +0100
@@ -39,8 +40,8 @@
     mp_dir = dir.path() +"/";
     if(QFile(mp_dir + "moneypenny.pro").exists())
       qDebug("Running from development directory tree.");
-    else
-     if(mp_dir.find("usr")>=1){ // may be /usr or /usr/local or /UNIONFS/usr on a KNOPPIX CD
+    else if(dir.absPath().startsWith("/usr") || dir.absPath().startsWith("/opt")){ 
+       // may be /usr or /usr/local or /UNIONFS/usr on a KNOPPIX CD
        mp_dir = mp_dir + "share/moneypenny/";
      }  
     QString fileName = mp_dir + MONEYPENNY_DATA_DIR + "/" + MONEYPENNY_HBCI_BANKDATA_FILE;
